resources/assets/utilities/RogueClient.js

```diff
@@ -2,7 +2,7 @@ import { RestApiClient } from '@dosomething/gateway';
 
 class RogueClient extends RestApiClient {
   getPosts(options) {
-    return this.get('posts', options);
+    return this.get('api/v3/posts', options);
```

Comment:
This is the initial call on page load where the campaign single page first grabs posts!

Reply:
Actually it's a little tricky! Here's how (I _think_) it works: 
- [`campaign_single.blade.php` passes `data-reviewing="true"` in the `wrapper`](https://github.com/DoSomething/rogue/blob/5cf34bb0adf4d88ff231a16a5b279bd20ba58930/resources/views/pages/campaign_single.blade.php#L8)
- The `data-reviewing="true"` then called the `WithReviewing` component to [`getPostsByStatus`](https://github.com/DoSomething/rogue/blob/5cf34bb0adf4d88ff231a16a5b279bd20ba58930/resources/assets/components/WithReviewing/index.js#L43) to populate the initial posts. 
- The line that you referenced up there gets new posts when you filter on the `CampaignSingle` page. 

Does that seem right?

Reply:
yea, i guess i'm just clarifying that you are just defining `getPosts` as a method on the `RogueClient` here. You aren't making an actually call. So somewhere you must be calling `RogueClient->getPosts()` which will really make the call. (sorry about that php syntax)

Reply:
oh yes!